src/node/game.js

```diff
@@ -1 +1,30 @@
-function Game() {}
+const Board = require('./board.js');
+const AllPlayers = require('./allPlayers');
+const TurnCounter = require('./turnCounter.js');
+
+function Game() {
+  const game = {
+    board: Board(),
+    players: AllPlayers(),
+    turnTracker: TurnCounter(),
+    start: () => runTheGame(game),
+  };
+  return game;
+}
+
+function runTheGame(game) {
+  while (game.players.hasNoWinner()) {
+    game.board.print();
+
+    const userMove = game.players
+      .whoHasTheTurn(game.turnTracker.getTurn())
+      .move(game.board);
```

Comment:
Nice usage of chaining methods! Just be careful, the moment I need to see `whoHasTheTurn` implementation to understand what it is returning, you should reconsider chaining at all:
``` js
const currentPlayer = game.players
      .whoHasTheTurn(game.turnTracker.getTurn())
currentPlayer.move(game.board);
```
Also you are actually needing it again in line :24, so a DRY (don't repeat yourself) here would be nice